README.md

```diff
@@ -1 +1,9 @@
-# tree-drive
+# Tree Drive
+
+This application is build by using the Django DRF framework for the API and use the ReactJs for the frontend.
+
+# Setup
+* Run `npm install` and then `npm run dev`.
```

Comment:
Add installation instructions for backend only

Reply:
Done